### DISCUSSION TITLE
Is this software able to be installed on Synology NAS?

### DISCUSSION THREAD

**Discussion by Dvalin21** [bigcapitalhq/bigcapital#137] [Q&A] [answered]
Would like to be able to install this software on my Synology NAS using Portainer. How compatible is this with Synology?  Thanks.

**Comment by abouolia** [comment, marked as answer]
I have not installed it on such technology, but you can install it on any linux-based distribution if I got any problem I'll follow and help in that.

**Reply by Dvalin21** [reply]
Ok thanks. I've shared this project over on Self Hosted subreddit. There is interest over there if your not already subscribed.
